tests/simple.sh

```diff
@@ -10,13 +10,15 @@ outPath=$(nix-store -rvv "$drvPath")
 
 echo "output path is $outPath"
 
+(! [ -w $outPath ])
```

Comment:
I don't think one needs the `(..)` subshell for this?

Reply:
It is needed. From the bash manpage under `set -e`: "The shell does not exit ... if the command's return value is being inverted with !".

Reply:
Oh, thanks! In that cases I think I need to fix some tests of mine on branches :)